ts/modules/marker-clusters.ts

```diff
@@ -0,0 +1,1948 @@
+     *         Zones on maps
+     *
+     * @type      {Array<ClusterZonesOptions>}
+     * @product   highcharts highmaps
+     * @optionparent plotOptions.scatter.marker.cluster.zones
+     */
+
+    /**
+     * Styled mode only. A custom class name for the zone.
+     *
+     * @sample highcharts/css/color-zones/
+     *         Zones styled by class name
+     *
+     * @type      {string}
+     * @optionparent plotOptions.scatter.marker.cluster.zones.className
+     */
+
+    /**
+     * Styles for the cluster zone.
+     *
+     * @see [StyleOptions](#plotOptions.scatter.marker.cluster.style)
+     * @extends   plotOptions.scatter.marker.cluster.style
+     * @type      {StyleOptions}
+     *
+     * @product   highcharts highmaps
+     * @optionparent plotOptions.scatter.marker.cluster.zones.style
+     */
+
+    /**
+     * The value where the zone starts.
+     *
+     * @type      {number}
+     *
+     * @product   highcharts highmaps
+     * @optionparent plotOptions.scatter.marker.cluster.zones.from
+     */
+
+    /**
+     * The value where the zone ends.
+     *
+     * @type      {number}
+     *
+     * @product   highcharts highmaps
+     * @optionparent plotOptions.scatter.marker.cluster.zones.to
+     */
+
+    /**
+     * The fill color of the cluster marker in hover state. When
+     * `undefined`, the series' or point's fillColor for normal
+     * state is used.
+     *
+     * @type      {Highcharts.ColorType}
+     * @optionparent plotOptions.scatter.marker.cluster.states.hover.fillColor
+     *
+     */
+
+    /**
+     * Options for the cluster data labels.
+     * @extends   plotOptions.series.dataLabels
+     * @type    {Highcharts.DataLabelsOptionsObject}
+     */
+    dataLabels: {
+        enabled: true,
+        format: '{point.clusterPointsAmount}',
+        verticalAlign: 'middle',
+        align: 'center'
+    }
+};
+
+(H.defaultOptions.plotOptions || {}).series = merge(
+    (H.defaultOptions.plotOptions || {}).series,
+    {
+        marker: {
+            cluster: clusterDefaultOptions
+        },
+        tooltip: {
+            /**
+             * The HTML of the cluster point's in the tooltip. Works only with
+             * marker-clusters module and analogously to
+             * [pointFormat](#tooltip.pointFormat).
+             *
+             * The cluster tooltip can be also formatted using
+             * `tooltip.formatter` callback function and `point.isCluster` flag.
+             *
+             * @sample highcharts/marker-clusters/grid
+             *         Format tooltip for cluster points.
+             *
+             * @sample maps/marker-clusters/europe/
+             *         Format tooltip for clusters using tooltip.formatter
+             *
+             * @type      {string}
+             * @apioption tooltip.clusterFormat
+             */
+            clusterFormat: '<span>Clustered points: ' +
+                '{point.clusterPointsAmount}</span><br/>'
```

Comment:
`@type` and `@default` are not needed, if it can be reflected from the default value, that follows.